src/View/ImageFusion/ImageFusionAxialView.py

```diff
@@ -167,44 +168,26 @@ def format_metadata(self, size: QtCore.QSize):
                 stylesheet = "QLabel { color : white; }"
             self.format_metadata_labels(stylesheet)
 
-    def image_display(self, color=False):
+    def image_display(self):
         """
         Update the image to be displayed on the DICOM View.
         """
-        if(color):
-            pixmaps = self.patient_dict_container.get("color_"+self.slice_view)
-            slider_id = self.slider.value()
-            image = pixmaps[slider_id]
-
-        else:
-            pixmaps = self.patient_dict_container.get(
-                "pixmaps_"+self.slice_view)
-            slider_id = self.slider.value()
-            image = pixmaps[slider_id]
+        pixmaps = self.patient_dict_container.get("color_"+self.slice_view)
+        slider_id = self.slider.value()
+        image = pixmaps[slider_id]
 
         label = QtWidgets.QGraphicsPixmapItem(image)
-        self.scene = QtWidgets.QGraphicsScene()
-        self.scene.addItem(label)
+        self.scene = GraphicsScene(label, self.horizontal_view, self.vertical_view)
 
-    def value_changed(self):
-        self.update_view(cut_line_color=True)
 
-    def update_view(self, zoom_change=False, cut_line_color=False):
+    def update_view(self, zoom_change=False):
         """
         Update the view of the DICOM Image.
         :param zoom_change: Boolean indicating whether the user wants 
         to change the zoom. False by default.
         """
-        if(cut_line_color):
-            self.image_display(color=cut_line_color)
-        else:
-            self.image_display()
-
-        if zoom_change:
-            self.view.setTransform(
-                QtGui.QTransform().scale(self.zoom, self.zoom))
-
-        self.view.setScene(self.scene)
+        super().update_view(zoom_change)
+        self.update_metadata()
 
     def update_metadata(self):
```

Comment:
The metadata on the axial view take too much space. You should consider formatting them like the DicomAxialView's metadata.

Reply:
Please elaborate, from what I can see - the ImageFusionAxialView metadata code is exactly the same DicomAxialView's.

Reply:
Before worrying about too much space always consider maintenance. When somebody comes back in 5 years, not having seen the code in that time, which layout will be the more easily understood? That is always the best code formatting to use.